segtypes/common/codesubsegment.py

```diff
@@ -72,14 +74,16 @@ def scan_code(self, rom_bytes, is_asm=False):
         # Process jumptable labels and pass them to spimdisasm
         self.gather_jumptable_labels(rom_bytes)
         for jtbl_label_vram in self.parent.jtbl_glabels_to_add:
-            rom_addr = self.ram_to_rom(jtbl_label_vram)
-            # TODO: what should we do when this is None?
-            if rom_addr is not None:
-                context_sym = self.text_section.addJumpTableLabel(
-                    jtbl_label_vram,
-                    isAutogenerated=True,
-                )
-                context_sym.name = f"L{jtbl_label_vram:X}_{rom_addr:X}"
+            rom_addr = self.get_most_parent().ram_to_rom(jtbl_label_vram)
+            sym = self.get_most_parent().create_symbol(jtbl_label_vram, True, type="jtbl_label", define=True)
+            sym.rom = rom_addr
```

Comment:
Why do we have to save the rom address back to the symbol here? For bss symbols, this will be wrong also I'm afraid :/ we probably need t omake ram_to_rom ignore bss_size